File: backend/components/metiers/administration_publique/guides/core/plugins/guides/guide_plugins.test.js
// guide_plugins.test.js – Tests unitaires JS pour guide_plugins.js
const { getPluginsGuide } = require('./guide_plugins');
describe('guide_plugins.js', () => {
  it('doit retourner un guide plugins complet', () => {
    const guide = getPluginsGuide();
    expect(guide.title).toBe('Guide Plugins 3D');
    expect(Array.isArray(guide.bestPractices)).toBe(true);
    expect(guide.bestPractices).toContain('Utiliser des plugins typés et documentés');
    expect(Array.isArray(guide.integrationSteps)).toBe(true);
    expect(guide.integrationSteps).toContain('Importer les plugins via le point d’entrée');
  });
});

File: backend/components/metiers/administration_publique/guides/core/plugins/guides/guide_plugins.test.py
"""
Tests ultra avancés pour guide_plugins.py (guides/core/plugins)
"""
from .guide_plugins import get_plugins_guide

def test_get_plugins_guide():
    guide = get_plugins_guide()
    assert isinstance(guide, dict)
    assert guide['title'] == 'Guide Plugins 3D'
    assert 'bestPractices' in guide
    assert 'integrationSteps' in guide
    assert 'Utiliser des plugins typés et documentés' in guide['bestPractices']
